check_patroni/types.py

```diff
@@ -35,44 +36,37 @@ class PatroniResource(nagiosplugin.Resource):
     def rest_api(self: "PatroniResource", service: str) -> Any:
         """Try to connect to all the provided endpoints for the requested service"""
         for endpoint in self.conn_info.endpoints:
-            try:
-                cert: Optional[Union[Tuple[str, str], str]] = None
-                verify: Optional[Union[str, bool]] = None
-                if endpoint[:5] == "https":
-                    if self.conn_info.cert is not None:
-                        # we can have: a key + a cert or a single file with key and cert.
-                        cert = self.conn_info.cert
-                    if self.conn_info.ca_cert is not None:
-                        # if cert is not None: this is the CA certificate
-                        # otherwise this is a ca bundle with root certificate
-                        # then some optional intermediate certificate and finally
-                        # the cerver certificate to validate the certification chain
-                        verify = self.conn_info.ca_cert
-                    else:
-                        if cert is None:
-                            # if cert is None we want to bypass https verification,
-                            # this is in secure and should be avoided for production use
-                            verify = False
-
-                _log.debug(
-                    f"Trying to connect to {endpoint}/{service} with cert: {cert} verify: {verify}"
-                )
+            cert: Optional[Union[Tuple[str, str], str]] = None
+            verify: Optional[Union[str, bool]] = None
+            if urlparse(endpoint).scheme == "https":
+                if self.conn_info.cert is not None:
+                    # we can have: a key + a cert or a single file with key and cert.
+                    cert = self.conn_info.cert
+                if self.conn_info.ca_cert is not None:
+                    # if cert is not None: this is the CA certificate
+                    # otherwise this is a ca bundle with root certificate
+                    # then some optional intermediate certificate and finally
+                    # the server certificate to validate the certification chain
```

Comment:
This comment still mentions `cert` whereas `ca_cert` is concerned here.

Reply:
cert relates to the variable with the certfile and key file. I will reword

Reply:
removed